Client/WebFilters/FilterListResourceDownloader.swift

```diff
@@ -265,11 +277,22 @@ public class FilterListResourceDownloader: ObservableObject {
     )
   }
   
+  /// This method allows us to enable selected lists by default for new users.
+  /// Make sure you use componentID to identify the filter list, as `uuid` will be deprecated in the future.
+  private func newFilterListEnabledOverride(for componentId: String) -> Bool? {
+    let componentIDsToOverride = [FilterList.mobileAnnoyancesComponentID]
+    
+    return componentIDsToOverride.contains(componentId) ? true : nil
+  }
+  
   /// Load filter lists from the ad block service
   private func loadFilterLists(from regionalFilterLists: [AdblockFilterList], filterListSettings: [FilterListSetting]) -> [FilterList] {
     return regionalFilterLists.map { adBlockFilterList in
       let setting = filterListSettings.first(where: { $0.uuid == adBlockFilterList.uuid })
-      return FilterList(from: adBlockFilterList, isEnabled: setting?.isEnabled ?? false)
+      return FilterList(from: adBlockFilterList,
+                        isEnabled: setting?.isEnabled
+                        ?? newFilterListEnabledOverride(for: adBlockFilterList.componentId)
+                        ?? false)
```

Comment:
I'm a little bit worried here: when I load filter lists for the first time I end up creating settings for all filter lists in the DB. I think what you do here is fine but just let's make sure this works fine.

Reply:
We decided to fix this separately or in the cookie-consent PR https://github.com/brave/brave-ios/pull/5839